sbirez/templates/sbirez/coversheet.html

```diff
@@ -376,7 +376,11 @@
               <dt>Phone</dt>
               <dd>{{ data.dod_coversheet.pi_info.pi_contact_info.pi_phone }}</dd>
               <dt>Fax</dt>
-              <dd>{{ data.dod_coversheet.pi_info.pi_contact_info.pi_fax }}</dd>
+              <dd>
```

Comment:
If you're just trying to keep the word "None" from appearing, there's a more concise django template filter: 
                <dd>{{ data.dod_coversheet.pi_info.pi_contact_info.pi_fax|default_if_none:"" }}</dd> 

https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#default-if-none